src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1008,6 +1008,149 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
             }
         }
 
+        public class IncludeSupport : JavascriptConversionExtension
+        {
+            public readonly string _alias;
+            public IncludeSupport(string inputAlias)
+            {
+                _alias = inputAlias;
+            }
+
+            public override void ConvertToJavascript(JavascriptConversionContext context)
+            {
+                var methodCallExpression = context.Node as MethodCallExpression;
+
+                if (methodCallExpression?.Method.Name != nameof(RavenQuery.Include))
+                    return;
+
+                if (methodCallExpression.Method.DeclaringType != typeof(RavenQuery))
+                    return;
+
+                if (methodCallExpression.Arguments.Count != 1)
+                    return;
+
+                context.PreventDefault();
+                
+                var writer = context.GetWriter();
+
+                //if not a lambda expression then
+                if (!methodCallExpression.Arguments[0].Type.BaseType.Name.Contains("LambdaExpression"))
+                {
+                    using (writer.Operation(methodCallExpression))
+                    {
+                        var arg = methodCallExpression.Arguments[0];
+
+                        if (arg is ConstantExpression { Value: string target1})
+                        {
+                            writer.Write("include(");
+                            writer.Write(target1);
+                            writer.Write(")");
+                            return;
+                        }
+                        throw new NotSupportedException("You can't use the include that way try: let _= RavenQuery.Include<T>(\"path\")");
```

Comment:
```suggestion
                        throw new NotSupportedException("You can't use the include that way try: let _ = RavenQuery.Include<T>(\"path\")");
```

also suggest the other overloads

Reply:
done